components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -261,4 +261,39 @@ function AutomaticPointsTable:calculatePointsForTournament(placement, manualPoin
 	return {}
 end
 
+function AutomaticPointsTable:sortData(pointsData, teams)
+	table.sort(pointsData,
+		function(a, b)
+			if a.totalPoints == b.totalPoints then
+				local t1 = teams[a.teamIndex]
+				local t1Name = t1.aliases[#t1.aliases]
+				local t2 = teams[b.teamIndex]
+				local t2Name = t2.aliases[#t2.aliases]
+				return t1Name < t2Name

```

Comment:
This looks weird. What's going on here?

Reply:
Sorts by names if two teams are tied in points